package.json

```diff
@@ -12,7 +12,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.3"
+    "react-scripts": "^4.0.3"
```

Comment:
is this update needed, @sachinsom93 ? Do you have any reason why we need this?

Reply:
@mtreacy002 , I didn't update it manually, it got updated when I use "npm install" at the time of project setup.

Reply:
@sachinsom93 We already have an issue open for updating the react-scripts version (#194). So, please revert these changes to the package.json and package-lock.json here.